loggers/zap/zap.go

```diff
@@ -0,0 +1,107 @@
+//Package zap provides a BaseLogger implementation for uber/zap
+package zap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-coldbrew/log/loggers"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type logger struct {
+	logger *zap.Logger
+	opt    loggers.Options
+	cfg    zap.Config
+}
+
+func (l *logger) Log(ctx context.Context, level loggers.Level, skip int, args ...interface{}) {
+
+	msg := fmt.Sprint(args...)
```

Comment:
we should add ctxfields to the logs as well - https://github.com/go-coldbrew/log/blob/613591fa528737e04db94d1fc53e84832bba2b1d/loggers/gokit/gokit.go#L29